content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -90,11 +109,17 @@ MTM-47731</td>
 <td>
 Inventory roles</td>
 <td>For users with inventory role access only, the performance for GET requests for operations has been improved. The feature is hidden behind a feature toggle which is already used for other improvements. The tenant option to enable/disable the toggle has the following payload: <code>"category": "configuration", "key": "acl.algorithm-version", "value": "OPTIMIZED"/"LEGACY"</code>. For details, see the <a href="https://www.cumulocity.com/api" class="no-ajaxy">Cumulocity IoT OpenAPI Specification</a>.</td>
-
 <td>
 MTM-42354</td>
 </tr>
 
+<tr>
+<td>Inventory roles</td>
+<td>Inventory roles algorithms with improved performance are now enabled on all tenants by default.
+In previous versions they were disabled, and customer needed to turn on feature per-tenant using tenant option or Operation team needed to turn on feature per environment.</td>

```

Comment:
```suggestion
In previous versions they were disabled, and customers needed to enable the feature for each tenant using a tenant option. Alternatively, the operations team could enable the feature per environment.</td>
```
maybe even remove the last sentence alltogether. it's more details than necessary for the side note here.

Reply:
There is definitely something wrong with this entry, need to double-check as this is not reviewed at all :-)